content/components/display/epaper_spi.md

```diff
@@ -28,41 +28,51 @@ display:
 
 ## Supported displays
 
-| Model name             | Manufacturer | Product Description                                        |
-| ---------------------- | ------------ | ---------------------------------------------------------- |
-| Spectra-E6             | Eink         | <https://www.eink.com/brand/detail/Spectra6>               |
-| 7.3in Spectra-E6       | Eink         | <https://www.eink.com/product/detail/EL073TF1U5>           |
-| Seeed-reTerminal-E1002 | Seeed Studio | <https://www.seeedstudio.com/reTerminal-E1002-p-6533.html> |
+| Model name             | Manufacturer | Product Description                                                                                                        |
+| ---------------------- |--------------|----------------------------------------------------------------------------------------------------------------------------|
+| Spectra-E6             | Eink         | <https://www.eink.com/brand/detail/Spectra6>                                                                               |
+| Seeed-reTerminal-E1002 | Seeed Studio | <https://www.seeedstudio.com/reTerminal-E1002-p-6533.html>                                                                 |
+| SSD1677                | Solomon      | <https://www.solomon-systech.com/product/ssd1677/>                                                                         |
+| Seeed-ee04-mono-4.26   | Seeed Studio | Seeed EE04 board with Waveshare 4.26" mono epaper. <https://www.seeedstudio.com/XIAO-ePaper-Display-Board-EE04-p-6560.html> |
 
 ## Configuration variables
 
 When using a model defining an integrated ESP32 display board most of the configuration such as the pins and dimensions will be set by default,
 but can be overridden if needed.
 
-- **model** (**Required**): The model of the ePaper display. See the table above for options.
+- **model** (**Required**): The model of the ePaper display. See the table above for options (case is not significant).
 - **cs_pin** (**Required**, [Pin Schema](/guides/configuration-types#pin-schema)): The CS pin. Predefined for integrated boards.
 - **dc_pin** (**Required**, [Pin Schema](/guides/configuration-types#pin-schema)): The DC pin. Predefined for integrated boards.
 - **busy_pin** (*Optional*, [Pin Schema](/guides/configuration-types#pin-schema)): The BUSY pin, if used.
 - **reset_pin** (*Optional*, [Pin Schema](/guides/configuration-types#pin-schema)): The RESET pin, if used.
   Make sure you pull this pin high (by connecting it to 3.3V with a resistor) if not connected to a GPIO pin.
+- **dimensions** (**Required**, dict): Dimensions of the screen, specified either as *width* **x** *height* (e.g `320x240`  )
+  or with separate config keys. For models with full pre-defined configuration this is optional and will be preset by
+  the model selected. The dimensions are specified in pixels, and the width and height must be greater than 0.
```

Comment:
Do all models require dimensions?

Reply:
Its not clear which ones do and which ones don't. Maybe add that to the table above ?